hi-ml-cpath/src/health_cpath/preprocessing/create_tiles_dataset.py

```diff
@@ -292,11 +295,11 @@ def main(slides_dataset: SlidesDataset, root_output_dir: Union[str, Path],
 
 
 if __name__ == '__main__':
-    from health_cpath.datasets.tcga_prad_dataset import TcgaPradDataset
+    from cpath.datasets.tcga_prad_private_dataset import TcgaPradPrivateDataset
```

Comment:
This dataset doesn't exist in this repository. Because this executable part here is meant as an example, I'd suggest leaving it as before.

Reply:
+1